governance.md

```diff
@@ -1,98 +1,137 @@
-# Tekton Bootstrap Governance
+# Tekton Governance
 
-The initial bootstrap committee will consist of 5 individuals and one governance facilitator who are core stakeholders and/or contributors.
+The Tekton Governance committee is the governing body of the Tekton open
+source project. It's an elected group that represents the contributors to
+the project, and has an oversight on governance and technical matters.
 
-Members are:
+## Tekton Governance Committee
 
-* [Andrew Bayer](https://github.com/abayer) (CloudBees)
-* [Vincent Demeester](https://github.com/vdemeester) (Red Hat)
-* [Christie Wilson](https://github.com/bobcatfish) (Google)
-* [Andrea Frittoli](https://github.com/afrittoli) (IBM)
-* [Jason Hall](https://github.com/ImJasonH) (Google)
+The Tekton governance committee consists of five elected individuals and one
+optional facilitator. The five seats are held for two year terms, staggered by
+one year: every year either two or three of the seats are up for election.
+
+### Current members
+
+Full Name         | Company   | GitHub  | Slack         | Elected On
+------------------|:---------:|---------|---------------|-----------
+Andrew Bayer      | CloudBees | [abayer](https://github.com/abayer)         | [@Andrew Bayer](https://tektoncd.slack.com/team/UJ6DJ4MSS)    | Bootstrap Committee
+Vincent Deemester | Red Hat   | [vdemeester](https://github.com/vdemeester) | [@vdemeester](https://tektoncd.slack.com/team/UHSQGV1L3)      | Bootstrap Committee
+Christie Wilson   | Google    | [bobcatfish](https://github.com/bobcatfish) | [@Christie Wilson](https://tektoncd.slack.com/team/UJ6DECY78) | Bootstrap Committee
+Andrea Frittoli   | IBM       | [afrittoli](https://github.com/afrittoli)   | [@Andrea Frittoli](https://tektoncd.slack.com/team/UJ411P2CC) | Feb 2020
+Jason Hall        | Google    | [ImJasonH](https://github.com/ImJasonH)     | [@Jason Hall](https://tektoncd.slack.com/team/UJ3MCRRRA)      | Feb 2020
+
+There is no designated facilitator at the moment, the responsibility is
+distributed across the five members of the committee.
```

Comment:
does it makes sense to mention the target date for the next election? i think it'll be me and @vdemeester and im guessing ~ feb 2021? (yikes!) np if we just want to leave that implied

Reply:
yes, I will add that